tools/i2nca/i2nca_agnostic_qc.xml

```diff
@@ -0,0 +1,99 @@
+<tool id="i2nca_agnostic_qc" name="i2nca Agnostic QC Report " version="@TOOL_VERSION@+galaxy@GALAXY_VERSION@">
+
```

Comment:
please use the Galaxy Language Server and reformat those tools, so that indentation matches etc ...